### PR TITLE
Add owner and submitter metadata

### PR DESCRIPTION
**Pull request by bfontecc007**
This adds the owner and submitter information to the build's metadata.

### CODE REVIEW

File: atomic_reactor/koji_util.py [hunk 1]
```diff
@@ -167,3 +167,15 @@ def tag_koji_build(session, build_id, target, poll_interval=5):
         raise RuntimeError('Task %s failed to tag koji build' % task_id)
 
     return build_tag
+
+
+def get_koji_task_owner(session, task_id, default=""):
+    if task_id is not None:
```
Comment (lcarva) [inline atomic_reactor/koji_util.py]: Maybe just `if task_id:` ?

File: atomic_reactor/plugins/exit_koji_import.py [hunk 2]
```diff
@@ -420,6 +421,12 @@ def get_build(self, metadata, worker_metadatas):
         if flatpak_source_info is not None:
             extra['image'].update(flatpak_source_info.koji_metadata())
 
+        if koji_task_id:
```
Comment (lcarva) [inline atomic_reactor/plugins/exit_koji_import.py]: Looks like `get_koji_task_owner` should be able to handle falsy `koji_task_id`.
This check is redundant in that case.

File: atomic_reactor/plugins/exit_koji_import.py [hunk 3]
```diff
@@ -420,6 +421,12 @@ def get_build(self, metadata, worker_metadatas):
...
+            koji_task_owner = get_koji_task_owner(self.session, koji_task_id, default=None)['name']
+        else:
+            koji_task_owner = None
+        extra['submitter'] = self.session.getLoggedInUser()['name']
```
Comment (vrutkovs) [inline atomic_reactor/plugins/exit_koji_import.py]: I don't think its correct - `getLoggedInUser` most likely would return koji account OSBS uses (`osbs`), not the original submitter.

See `_get_koji_submitter` in `exit_sendmail.py`

Comment (lcarva) [reply atomic_reactor/plugins/exit_koji_import.py]: Actually, submitter is the one who submitted the build via content generator API (osbs), and owner is the user that started the task (me, for example). The owner is used to populate the "Built by" field in the UI, for example. The submitter field is simply added for tracking purposes, AFAIK it's not actually actively used.

@breillyr, please confirm.

File: atomic_reactor/plugins/exit_koji_import.py (hunk 2 above)
Comment (bfontecc007) [reply atomic_reactor/plugins/exit_koji_import.py]: I added that guard when I added the index by 'name'. You can't index None.

File: atomic_reactor/plugins/exit_koji_import.py (hunk 3 above)
Comment (twaugh) [reply atomic_reactor/plugins/exit_koji_import.py]: Ah! Yes, now I see. "owner" is described in the [documentation](https://docs.pagure.org/koji/content_generator_metadata/).

I don't see "submitter" described anywhere except for the description of [this issue](https://pagure.io/koji/issue/299#), but from the story description the code looks right.

**Comment by bfontecc007** [comment]
I added a change to the docs.

**Comment by lcarva** [comment]
Updated release notes draft:
> - Koji build metadata contains owner and submitter

File: atomic_reactor/plugins/exit_koji_import.py (hunk 2 above)
Comment (sosiouxme) [reply atomic_reactor/plugins/exit_koji_import.py]: @bfontecc007 why put `default=None` then if you know you'll get an exception when you actually get `None` back? Is that desirable if your task id comes up with nothing?

Comment (bfontecc007) [reply atomic_reactor/plugins/exit_koji_import.py]: @sosiouxme I don't really remember in great detail but maybe I thought it was an easier to interpret exception than the one you'd get if you tried to index string '' by 'name.'

Comment (twaugh) [reply atomic_reactor/plugins/exit_koji_import.py]: @sosiouxme is right, that doesn't make any sense.

It looks to me like it should be `get_koji_task_owner()` that does the ['name'] look-up, not the caller of that function.

**Pull request merged**